clients/drcachesim/tests/burst_syscall_inject.cpp

```diff
@@ -167,7 +171,14 @@ write_system_call_template(void *dr_context)
     write_trace_entry(
         writer,
         make_memref(READ_MEMADDR_GETTID, TRACE_TYPE_READ, opnd_size_in_bytes(OPSZ_PTR)));
+    write_trace_entry(writer,
+                      make_marker(TRACE_MARKER_TYPE_SYSCALL_TRACE_END, SYS_gettid));
 
+    // Write footer.
+    dynamorio::drmemtrace::trace_entry_t thread_exit = {
+        dynamorio::drmemtrace::TRACE_TYPE_THREAD_EXIT, 0, { /*tid=*/1 }
+    };
+    write_trace_entry(writer, thread_exit);
```

Comment:
This was discussed in a previous PR (https://github.com/DynamoRIO/dynamorio/pull/6496/files/fec4728deac3d1ba50f6c4ef2c46d25db94d260a#r1418372222 and https://github.com/DynamoRIO/dynamorio/pull/6496#discussion_r1419848311): we decided to defer zip file support because the record_reader_t support wasn't available then, and for initial experiments non-zipfile would be fine.

I'd prefer to take on the move to zipfile separately from this PR.

Reply:
OK.